### ISSUE TITLE
Implement Do Not Track header

### ISSUE THREAD

**Issue by SebastianZimmeck** [privacy-tech-lab/optmeowt-browser-extension#44]
At least one major publisher seems to interpret Do Not **Track** signals as Do Not **Sell** signals. So, it may be worthwhile to [implement Do Not Track](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/DNT) as well. Essentially, the Do Not Track setting would exactly mirror whatever the user has selected for Do Not Sell.

(cc'ing @davebaraka and @rgoldstein01)

**Comment by kalicki1** [comment]
This was a relatively simple implementation: we have already built the infrastructure for sending headers to every site we visit. [Here is the DNT (Do Not Track) signal](https://github.com/privacy-tech-lab/optmeowt-browser-extension/blob/431fbc687fefbe39c0fb711293444deb5782ce54/src/background.js#L28) in the extension. I also have a photo of the headers side by side in the Chrome console below: 

<img width="275" alt="Screen Shot 2020-07-08 at 11 47 28 PM" src="https://user-images.githubusercontent.com/60238705/86995277-8c28b080-c176-11ea-9354-762fc48aafb0.png">

I took a look at [Mozilla's documentation](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/DNT) regarding the DNT signal, and it looks like the official standard dictates the header is supposed to be sent to every site, even if it it only contains a `null` or `0` value. In reality, it looks like both Chrome and Firefox only ever send the signal when the browser-wide option for the signal is enabled, similar to how we have been operating with our Do Not Sell signal. 

@SebastianZimmeck, to me it feels like this is all there is to this particular issue. I did not upload directly to the master branch for the sake of keeping the code changes with the issue, though I am ready to merge this back in if you feel it is all set as well.
